AprilTagTrackers/Connection.cpp

```diff
@@ -78,14 +78,33 @@ void Connection::Connect()
 
 std::istringstream Connection::Send(std::string lpszWrite)
 {
+    int fd;
+
+    fd = open(lpszPipename, O_WRONLY);
+    write(fd, lpszWrite.c_str(), strlen(lpszWrite.c_str()) + 1);
+    close(fd);
+
+    fd = open(lpszPipename, O_RDONLY);
+    cbRead = read(fd, chReadBuf, BUFSIZE * sizeof(char));
+    close(fd);
+
+    std::cout << chReadBuf << std::endl;
+    chReadBuf[cbRead] = '\0';
+    std::string rec = chReadBuf;
+    std::istringstream iss(rec);
+
+    return iss;
+
+    /*

```

Comment:
Use `#ifdef _WIN32` for conditional compilation.

Reply:
I'm leaving all that until I get something functional ;)